### PR TITLE
Add Some api

### PR DESCRIPTION
**Pull request by eyworldwide**
- fix: color of spriteRenderer
- feat: add `pixelsPerUnit` api
- feat: add `setLayer` method

### CODE REVIEW

File: src/LottieAnimation.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ export class LottieAnimation extends Script {
 	private _clip: TypeAnimationClip;
 	private _clipEndCallbacks: Object = {};
 	private _autoPlay: boolean = false;
+	private _pivotVector: Vector2 = new Vector2();
```
Comment (cptbtptpbcptdtptp) [inline src/LottieAnimation.ts:30]: Static can be used here.

**Review by cptbtptpbcptdtptp — CHANGES_REQUESTED** [review]
Hope to change _pivotVector to static type.
Line 30 of ‘src/LottieAnimation.ts’.

File: src/LottieAnimation.ts (hunk 1 above)
Comment (cptbtptpbcptdtptp) [reply src/LottieAnimation.ts:30]: image
This is how the sprite sets the pivot.

Comment (eyworldwide) [reply src/LottieAnimation.ts]: Fine.

**Review by singlecoder — APPROVED** [review]
+1

**Pull request merged**